docker-compose.yml

```
@@ -17,10 +26,15 @@ services:
     ports:
       - 1337:1337
     environment:
-      - DB_HOST=db:3306
-      - DB_NAME=${MYSQL_DATABASE}
-      - DB_USER=root
-      - DB_PASSWORD=${MYSQL_ROOT_PASSWORD}
+      - MYSQL_HOST=db
+      - MYSQL_DB=clock_of_change
+      - MYSQL_USER=root
+      - MYSQL_PASS=123456
+      - MAIL_HOST=mailhog
+      - MAIL_PORT=1025
+      - MAIL_USER=mailer
+      - MAIL_PASS=123456
     depends_on:
       - db
-    command: "npm run start"
+      - mailhog
+    command: bash -c "sleep 5 && db-migrate up && npm run start"
```

```suggestion
    command: bash -c "wait-on tcp://db:3306 && db-migrate up && npm run start"
```
@matthib I don't think `sleep` is a great solution. I had multiple timing issues. Use https://www.npmjs.com/package/wait-on.

---

@roschaefer See my last commit: https://github.com/Human-Connection/Clock-of-Change-API/pull/18/commits/6db1cc7305f6e8ad0fb21d6a01358646e9546f15